features/webcryptoapi.yml

```
@@ -1,7 +1,12 @@
-draft_date: 2024-08-12
-name: Web Cryptography API
-description: TODO
-spec: https://w3c.github.io/webcrypto/
+name: Web Cryptography
+description: The Web Cryptography API provides utilities and primitives for performing basic cryptographic operations.
```

I have an allergy to the word "basic" (basic for whom, exactly? Not me!). I'm suggesting "low-level" here as probably capturing the same idea and including some examples to fill it out:

```suggestion
description: The Web Crypto API performs low-level cryptographic operations, such as encryption, decryption, and signature verification. Also known as the Web Cryptography API.
```

---

@autonome This wasn't accepted and the existing text still stands. Were you going to revisit this yourself?

---

Nope, i just missed it and pushed changes after - just updated now, but w/ the aka reversed until we settle on final name.

---

and yes, +1 on nobasicness, thanks for pointing that out!

---

fixed